pbst.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * @file
+ * @version 0.1
+ * @copyright 2017 CN-Consult GmbH
+ * @author Jens Stahl <jens.stahl@cn-consult.eu>
+ *
+ * License: Please check the LICENSE file for more information.
+ */
+
+require_once ("vendor/autoload.php");
+foreach (glob(getcwd()."/commands/*command.php") as $filename)
+{
+    require_once ($filename);
+}
+require_once ('profitbricksapi/profitbricksapi.php');
+require_once ('profitbricksapi/datacenter.php');
+require_once ('profitbricksapi/virtualmachine.php');
+require_once ('profitbricksapi/virtualdisk.php');
+require_once ('profitbricksapi/snapshot.php');
+
+use Symfony\Component\Console\Application;
+
+$application = new Application();
+$application->add(new \PBST\Command\SnapshotListCommand());
```

Comment:
I would like to add a bit coolness to the commands entrypoint and replace these add-operations with a loop:
```
$application = new Application();
foreach(array("SnapshotList", "SnapshotDelete", ...) as $command)
{
    $command = "\\PBST\\Command\\".$command;
    $application->add(new $command());
}
```

If something goes wrong with the autoloading stuff which might be not able to load the namespaced classes,
this replacement will always work.

[ As fallback-strategy, the require_once-repetitions can also replaced with a normal glob-loop ]